bootstrap_sdk

```diff
@@ -83,6 +83,87 @@ mkdir -p "${ROOT_OVERLAY}/tmp"
 chmod 1777 "${ROOT_OVERLAY}/tmp"
 cp "${BUILD_LIBRARY_DIR}/toolchain_util.sh" "${ROOT_OVERLAY}/tmp"
 
+
+# Stage 1 uses "known-good" ebuild repos (both coreos-overlay and portage-stable)
+#  to build a minimal toolchain (USE="-*") for stage 2.
+#
+# No package updates must happen in stage 1, so we use
+# - the portage-stable repo included with the current SDK (from the SDK chroot's
+#    /var/lib/gentoo/repos/gentoo), and
+# - the coreos-overlay repo version listed in the current SDK's manifest.
+#    Unfortunately coreos-overlay is not included in the SDK chroot at all; this
+#    will be addressed in the future.
+#
+# "Current SDK" refers to the SDK we entered with 'cork enter', i.e. the SDK
+#   we run ./bootstrap_sdk in.
+#
+# Using ebuilds from the above mentioned sources will ensure that stage 1 builds
+#  a minimal stage 2 from known-good ebuild versions - the same ebuild versions
+#  were used to build the very SDK ware currently in.
```

Comment:
```suggestion
#  were used to build the very SDK we're currently in.
```

But even after fixing the typo this is confusing, "we" are not in an SDK. Maybe "were used to build the SDK that is used to run the bootstrap_sdk script"?